mod_conf_sql.html

```diff
@@ -195,6 +213,11 @@
   proftpd -c sql://foo:bar@localhost/db:proftpd/ctxt:ftpctxt/conf:ftpconf/map:ftpmap/base_id=<i>baseid</i>
 </pre>
 
+<p> for sqlite dbuser and dbpass are note used, use dbserver for file path and use \ in place of / for the file path : </p>
+<pre>
+  proftpd -c sql://nouser:nopass@\path\to\sqlite.db/db:sqlite/ctxt:ftpctxt/conf:ftpconf/map:ftpmap
```

Comment:
or while be obliged to rewrite the parsing code of the uri and change drasticly the uri forms...

them use url args for database argument... and hostname and path for the database name...

idealy the url format may be : 
`sql://(/path/to/dabasefile.db|databasename)?dbpass=pass&dbuser=user&dbhost=hostname&dbctxt=ftpctxt&conf=ftpconf&map=ftpmap`

exemple for sqlite:
`sql:///path/to/sqlitefile.db?dbctxt=ftpctxt&conf=ftpconf&map=ftpmap`

exemple for mysql and pgsql:
`sql://databasename?bbuser=user&dbpass=pass&dbhost=hostname&dbctxt=ftpctxt&conf=ftpconf&map=ftpmap`

but this while break compatibility while old version configuration format...